sse2neon.h

```diff
@@ -1773,10 +1773,12 @@ FORCE_INLINE __m128 _mm_div_ss(__m128 a, __m128 b)
 
 // Free aligned memory that was allocated with _mm_malloc.
 // https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_free
+#if !defined(_WIN32)
```

Comment:
Can you define `SSE2NEON_ALLOC_DEFINED` right after `#include <stdlib.h>`? That is, I expect the following:
```c
#include <stdint.h>
#include <stdlib.h>

#if defined(_WIN32) || defined(_WIN64)
/* Definitions for _mm_{malloc,free} are provided by <malloc.h>
 * from both MinGW-w64 and MSVC.
 */
#define SSE2NEON_ALLOC_DEFINED
#endif
...
#if !defined(SSE2NEON_ALLOC_DEFINED)
FORCE_INLINE void _mm_free(void *addr)
{
    free(addr);
}
#endif


```

Reply:
Done with latest push. I only tested for _WIN32 because it is defined for all architectures.